mbuild/formats/lammpsdata.py

```diff
@@ -142,7 +142,6 @@ def write_lammpsdata(structure, filename, atom_style='full',
     box = Box(lengths=np.array([0.1 * val for val in structure.box[0:3]]),
               angles=structure.box[3:6])
     # Divide by conversion factor
-    box.lengths /= sigma_conversion_factor
```

Comment:
Doesn't look like it was documented anywhere. Just to have it on record, it looks like this PR also fixes an issue with the lammps writer where the box dimensions were doubly divided by the conversion factor.

Reply:
Yeah, I think before, doing `box.lengths /= sigma_conversion_factor` does not automatically update `box.maxs`, so we have to do it twice.

Reply:
Ah. Ok that makes sense.